cypress/integration/notificationDom.spec.js

```diff
@@ -9,27 +9,30 @@ Cypress.Commands.add('hasNotification', (selector, posTop, posRight) => {
 
 describe('Notifications app', () => {
   beforeEach(() => {
+    cy.clock();
     cy.visit('/');
-    cy.get('.notification.success').as('success');
-    cy.get('.notification.error').as('error');
-    cy.get('.notification.warning').as('warning');
+  });
+
+  afterEach(() => {
+    cy.clock().then(clock => clock.restore());
   });
 
   it('should print `success` message', () => {
-    cy.hasNotification('@success', 10, 10);
+    cy.hasNotification('.notification.success', 10, 10);
   });
 
   it('should print `error` message', () => {
-    cy.hasNotification('@error', 150, 10);
+    cy.hasNotification('.notification.error', 150, 10);
   });
 
   it('should print `warning` message', () => {
-    cy.hasNotification('@warning', 290, 10);
+    cy.hasNotification('.notification.warning', 290, 10);
   });
 
   it('messages should disappear in 2 seconds', () => {
-    cy.get('@success', { timeout: 2000 }).should('not.visible');
-    cy.get('@error', { timeout: 2000 }).should('not.visible');
-    cy.get('@warning', { timeout: 2000 }).should('not.visible');
+    cy.tick(2200)
```

Comment:
The 'cy.tick()' function is used to advance the clock, but it's not clear if the notifications are created with a delay that matches this tick. Ensure that the notifications are indeed set to disappear after 2 seconds.